src/zos-io.cc

```diff
@@ -700,6 +714,62 @@ int __disableautocvt(int fd) {
   return fcntl(fd, F_CONTROL_CVT, &req);
 }
 
+int __ccsid_new_file() {
+  char* encode_file_new = getenv("_ENCODE_FILE_NEW");
+
+  int ccsid = 819;
+
+  if (encode_file_new)
+    if (strcmp(encode_file_new, "IBM-1047") == 0) {
+      ccsid = 1047;
+    } else if (strcmp(encode_file_new, "BINARY") == 0) {
+      // Set the file descriptor to binary mode
+      ccsid = FT_BINARY;
+    }
+
+  return ccsid;
+}
+
+int __tag_new_file_filename(int fd, std::string filename) {
```

Comment:
Do we still need __tag_new_file below?

Reply:
I think it is used somewhere else, will have to look it up